src/service/localization.js

```diff
@@ -146,9 +141,9 @@ export class LocalizationService {
    * @return {string|null}
    */
   getLocalizedString(localizedStringId, elementToUse = undefined) {
-    const languageCodes = elementToUse
-      ? this.getLanguageCodesForElement_(elementToUse)
-      : this.rootLanguageCodes_;
+    const languageCodes = this.getLanguageCodesForElement_(
+      elementToUse || this.element_
```

Comment:
Nit: You can specify `elementsToUse = this.element_` directly in the parameters list

Reply:
Oh cool!